wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md

```diff
@@ -15,7 +18,7 @@ Easy Method (don't have any local changes):
 3.  Go into editor. Make sure the "Web" menu is enabled (not greyed out).
 4.  Change the Artist/Title info.
 5.  **Do NOT exit the editor.**
-6.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+6.  Use **File > Open...** to change the Artist/Title in other difficulties.
```

Comment:
idk, but since these are buttons, ``6.  Use `File` > `Open...` to change the Artist/Title in other difficulties.``